src/overlays/actors/ovl_Eff_Dust/z_eff_dust.h

```diff
@@ -5,12 +5,32 @@
 #include "global.h"
 
 struct EffDust;
+typedef void (*EffDustActionFunc)(struct EffDust *, GlobalContext *);
+typedef void (*EffDustDrawFunc)(Actor *, GlobalContext *);
 
 typedef struct EffDust {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x41C];
+    /* 0x014C */ f32 distanceTraveled[0x40]; //!< For each particle. Normalized. From 0.0f to 1.0f
+    /* 0x024C */ Vec3f initialPositions[0x40]; //!< Array of position for each dust particle.
+    /* 0x054C */ u8 index;
+    /* 0x054D */ u8 life; //!< Only considered if actor.params is 2, 3 or 4.
+    /* 0x0550 */ f32 dx; //!< Normalized. 0.0f to 1.0f
+    /* 0x0554 */ f32 dy; //!< Normalized. 0.0f to 1.0f
+    /* 0x0558 */ f32 dz; //!< Normalized. 0.0f to 1.0f
+    /* 0x055C */ f32 scalingFactor;
+    /* 0x0560 */ EffDustActionFunc updateFunc;
+    /* 0x0560 */ EffDustDrawFunc drawFunc;
 } EffDust; // size = 0x0568
 
+// EffDust's params.
+typedef enum EffDustType {
```

Comment:
There is no need for the comment imo since you gave the enum type a proper name. Also the `enum` itself doesn't need to be named so we usually omit it and just leave the name for the `typedef` (at the bottom of the enum)
```suggestion
typedef enum {
```